requirements.txt

```diff
@@ -43,7 +43,7 @@ tensorboardX==2.1
 tokenizers>=0.8.0
 torchtext>=0.5.0
 tornado==6.0.4
-tqdm~=4.36.1
+tqdm~=4.38.0
```

Comment:
is it from previous merged commits or new one? if it's previously merged commit please rebase on the latest master. if not, I would suggest we separate this into 2 different pull requests.

Reply:
yep... it's a previous merge... will try to rebase 👍